src/Jsonize.Test/Program.cs

```diff
@@ -23,6 +25,18 @@ public static void Main(string[] args)
             Console.ReadLine();
         }
 
+        [SetUp] 
+        public void Init()
+        {
+            File.WriteAllText(TEXTFILE, "");
+        }
+
+        [TearDown] 
+        public void Cleanup() 
+        {
+            File.Delete(TEXTFILE);
```

Comment:
Tests should never leave artifacts behind. So either you delete the text file when all tests have finished or you add `*.txt` files to your `.gitignore` file.